js/src/pages/price-benchmark/price/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import useAdsCurrency from '~/hooks/useAdsCurrency';
+import './index.scss';
+
+/**
+ * Price component to display a formatted price value with optional highlighting.
+ *
+ * @param {Object} props - The component props.
+ * @param {number} props.amount - The price value to be formatted and displayed.
+ * @param {boolean} [props.highlight=false] - Whether to apply a highlight style to the price.
+ * @return {JSX.Element} The rendered price component.
+ */
+const Price = ( { amount, highlight = false } ) => {
+	const { formatAmount } = useAdsCurrency();
```

Comment:
Eventually, it would be nice to have some jest tests for this component that confirm this is working as intended with different currency types.

Reply:
Added tests although the test are mostly to test the isNaN functionality and highlight.